TombEngine/Game/control/trigger.cpp

```diff
@@ -118,48 +118,60 @@ int GetSwitchTrigger(ItemInfo* item, short* itemNos, int attatchedToSwitch)
 	return 0;
 }
 
-int SwitchTrigger(short itemNumber, short timer)
+bool SwitchTrigger(short itemNumber, short timer)
```

Comment:
This should not be part of this PR.
Please separate the Airlock Switch from the Trigger Refactor!

Reply:
Yes, Sezz suggested that could be a good chance to do the update, but I neither see any problem postponing that for after the airlock and switches get merged.

I've push another commit that turn the bool back to int as originally was.